hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/producer/BrokerMessagesAsyncProducer.java

```diff
@@ -0,0 +1,11 @@
+package pl.allegro.tech.hermes.frontend.producer;
+
+import pl.allegro.tech.hermes.frontend.metric.CachedTopic;
+import pl.allegro.tech.hermes.frontend.publishing.message.Message;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+public interface BrokerMessagesAsyncProducer {
```

Comment:
Do we need `Async` in the interface name ? You don't need to specify any internals of the producer here because it's just an abstraction :)

Reply:
I know it'll conflict with already existing interface, but maybe we could give a name suggesting that it's for out healthchecking purposes ? E.g `HealthcheckingBrokerMessagesProducer` or `BrokerAvailabilityMessagesProducer` ?

Reply:
I'll name it simply BrokerMessageSProducer, thus it will not conflict with existing interface and beside this, it has method signature `publishMessages(@NotNull CachedTopic topic, List<Message> messages, long timeoutMs);` Is it ok?